jax/_src/export/_export.py

```diff
@@ -1013,6 +1013,7 @@ def _check_lowering(lowering) -> None:
     "lapack_ssyevd_ffi", "lapack_dsyevd_ffi", "lapack_cheevd_ffi", "lapack_zheevd_ffi",
     "lapack_sgeev_ffi", "lapack_dgeev_ffi", "lapack_cgeev_ffi", "lapack_zgeev_ffi",
     "lapack_sgesdd_ffi", "lapack_dgesdd_ffi", "lapack_cgesdd_ffi", "lapack_zgesdd_ffi",
+    "lapack_sgesvd_ffi", "lapack_dgesvd_ffi", "lapack_cgesvd_ffi", "lapack_zgesvd_ffi",
```

Comment:
To include this here we would need to add backwards compatibility tests, so let's skip that for now! (i.e. please remove this line :D)